src/vivarium_gates_mncnh/components/children.py

```diff
@@ -14,127 +24,105 @@ class NewChildren(Component):
     ##############
 
     @property
-    def sub_components(self) -> List[str]:
-        return [self.lbwsg]
+    def columns_created(self) -> list[str]:
```

Comment:
Changes in this file are for the refactor to the pregnancy component since the NewChildren component was more a class that stroed methods and it is now a proper subcomponent.